src/pages/Login/index.js

```diff
@@ -9,12 +9,13 @@ import { useDispatch } from 'react-redux';
 import { api } from 'services/api';
 import { useLoginMutation } from 'services/auth/auth';
 import { setLoggedInUser } from 'utils/auth';
+import VideoContainer from './VideoContainer';
 import useAuth from 'hooks/useAuth';
 import useTranslation from 'hooks/useTranslation';
 import routesPaths from 'routes/routesPaths';
 import Input from 'components/form/Input';
 import Button from 'components/common/Button';
-
+import './styles.css';
```

Comment:
as you have a big component and 2 stylesheets you can consider split this in 2 separate components, <Login /> and <LoginForm />. So inside Login you can call LoginForm and move form.css to LoginForm

Reply:
will do